packages/cssnano-preset-default/src/index.js

```diff
@@ -49,6 +51,9 @@ const defaultOpts = {
     normalizeCharset: {
         add: false,
     },
+    cssDeclarationSorter: {
+        customOrder: path.join(__dirname, 'safe-order.json'),
```

Comment:
The declarations cannot be sorted based on a sort order defined in a `json` file. They need to respect the order of declaration in the CSS, else the precedence will be different and the CSS will render differently in the browser.

Reply:
Note that I'm leaving out the declarations that need to be kept in their original order, so this will work. Do you have an example that will fail?